.github/workflows/lint-and-build.yml

```diff
@@ -31,6 +31,10 @@ env:
   GITHUB_HEAD_REPOSITORY: ${{ github.event.pull_request.head.repo.full_name }}
   GITHUB_EXCLUDE_BUILD_NUMBER: ${{ inputs.excludeBuildNumber }}
   UV_NO_SYNC: true # Avoid accidentally pulling in dependency-groups with uv run
+  # https://github.com/opencv/opencv-python#source-distributions
+  # Allows building OpenCV on Windows ARM64
+  # https://github.com/opencv/opencv-python/issues/1092#issuecomment-2862538656
+  CMAKE_ARGS: "-DBUILD_opencv_dnn=OFF -DENABLE_NEON=OFF"
```

Comment:
Thanks to https://github.com/opencv/opencv-python/issues/1092#issuecomment-2862538656